velox/common/base/SpillStats.h

```diff
@@ -95,6 +95,10 @@ struct SpillStats {
   std::string toString() const;
 };
 
+auto format_as(SpillStats stats) {
```

Comment:
Is format_as something expected by fmt? The naming doesn't match Velox naming convention. Perhaps, add a comment to explain why this is needed.